src/components/Footer.js

```diff
@@ -23,8 +21,7 @@ const Footer = props => (
           <a href="https://www.linkedin.com/in/gandrade/" target="_blank">
             Gaby Andrade
           </a>
-        </dd>
```

Comment:
removing a `<dd />` here so that this is one line and separated by a comma vs. two separated cells